### PR TITLE
Pushing Live

### PR DESCRIPTION
**Pull request by DuncanmaMSFT**

### CODE REVIEW

**Comment by DuncanmaMSFT** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[pages/docfx.json](https://github.com/MicrosoftDocs/SamplesPages/blob/master/pages/docfx.json) | :white_check_mark:Succeeded | |
[pages/pagetypes/conceptual.md](https://github.com/MicrosoftDocs/SamplesPages/blob/master/pages/pagetypes/conceptual.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/pages/pagetypes/conceptual?branch=pr-en-us-157) |
[pages/pagetypes/hubpage.md](https://github.com/MicrosoftDocs/SamplesPages/blob/master/pages/pagetypes/hubpage.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/pages/pagetypes/hubpage?branch=pr-en-us-157) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C30%5Ccc9c7779-9b06-9129-76ea-0d1ec96a2a92%5CPullRequest%5C201811302239545570-157%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wqHPbR9PQ0PCM4vjms9fX3y8AhWG4ni8ymyXs%2F2h4M4%3D&st=2018-11-30T22%3A36%3A11Z&se=2018-12-31T22%3A41%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: pages/docfx.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     },
     "fileMetadata": {},
     "template": [],
-    "dest": "pages"
+    "dest": "pages",
+    "markdownEngineName": "markdig"
```
Comment (EisenbergEffect) [inline pages/docfx.json]: Booyah!!!

Comment (adkinn) [reply pages/docfx.json]: Hah!

Comment (EisenbergEffect) [reply pages/docfx.json]: Happy Friday to my old team!

**Pull request merged**